en/docs/install-and-setup/setup/reference/default-product-ports.md

```diff
@@ -1,13 +1,86 @@
 # Default Product Ports
 
-This page describes the default ports that are used for each WSO2 product when the [port offset]({{base_path}}/install-and-setup/setup/deployment-best-practices/changing-the-default-ports-with-offset/#configuring-the-port-offset) is 0.
+This page describes the default ports used by each runtime of WSO2 API Manager.
 
 !!! Attention
     **Note** that it is recommended to disable the HTTP transport in an API Manager production setup. Using the `Bearer` token over HTTP is a violation of the OAuth specification and can lead to security vulnerabilities.
 
+!!! Note
+    If you change the default runtime ports with a port offset, most of the runtime ports change automatically based on the offset.
 
--   [Product-specific ports](#product-specific-ports)
--   [Disabling HTTP Transports](#disabling-http-transports)
+## WSO2 API-M ports
+
+Listed below are the ports used by the API-M runtime when the [port offset]({{base_path}}/install-and-setup/setup/deployment-best-practices/changing-the-default-ports-with-offset/#configuring-the-port-offset) is 0.
```

Comment:
@nilminiwso2 need to add webhooks inbound endpoints. 9021 (http) and 8021 (https)

Reply:
Will send a new PR for these.